README.md

```diff
@@ -149,11 +149,21 @@ git machete advance
 Useful for merging the child branch to the current branch in a linear fashion (without creating a merge commit).
 
 ### GitHub integration
+
+Annotate the branches with GitHub PR numbers: <br/>
+```shell script
+git machete github anno-prs
+```
+
+Create the PR, using the parent branch from `.git/machete` as the base: <br/>
```

Comment:
```suggestion
Create the PR, using the upstream (parent) branch from `.git/machete` as the base: <br/>
```

Reply:
The reason why I still keep this double naming (_up/downstream_, _parent/child_) is that:
* the former works better with "navigation" in `go`/`show` commands (it's more natural to _go up_ rather than _go (to) parent_ BUT
* the latter is more established nomenclature when speaking about trees in theoretical CS/math.

CC @BartoszSiwinski

Reply:
Neither is free of naming conflicts tho:
* _upstream_ (although not _downstream_) is used by git CLI to mean _remote for the given branch_, see https://virtuslab.github.io/tips/#git/upstream
* _parent_ (although not _child_) is used for _parent commits_, as in e.g. the output of `git cat-file -p HEAD`.